doc/user/content/sql/appendix-cluster-sizes.md

```diff
@@ -7,8 +7,40 @@ menu:
     weight: 900
 ---
 
+## Default Cluster Sizes
+
 {{% self-managed/materialize-cluster-sizes %}}
 
+## Custom Cluster Sizes
+
+When installing the Materialize Helm chart, you can override the [default
+cluster sizes and resource allocations](/sql/appendix-cluster-sizes/). These
+cluster sizes are used for both internal clusters, such as the `system_cluster`,
+as well as user clusters.
+
+{{< tip >}}
+
+In general, you should not have to override the defaults. At minimum, we
+recommend that you keep the 25-200cc cluster sizes.
+
+{{</ tip >}}
+
+```yaml
+operator:
```

Comment:
And, ... I'm guessing that if using terraforms, people would set these via https://github.com/MaterializeInc/terraform-aws-materialize?tab=readme-ov-file#input_helm_values ?

Reply:
Yup!

Reply:
I think we hide this from our documentation (which is probably good) but if you pull down any of the sample values or defaults it'll be there for people to change.

Reply:
Something to consider for the future is I'm familiar with adding some blurb about "If you need guidance on ..., <X professional services> offer ...." kind of a blurbs.  We're not there yet but custom cluster sizing might be one of those.